packages/datadog-plugin-tedious/test/index.spec.js

```diff
@@ -22,6 +30,7 @@ describe('Plugin', () => {
       let config
 
       beforeEach(() => {
+        console.log('ABOUT TO LOAD TEDIOUS', version, process.version)
         return agent.load('tedious').then(() => {
           tds = require(`../../../versions/tedious@${version}`).get()
```

Comment:
According to the failing test output, this line is NOT the first place where `tedious` is required. It looks like this happens somewhere earlier in the installation phases, before these tests even run.